### DISCUSSION TITLE
[Console] NaN in Pages Component In Variable Section

### DISCUSSION THREAD

**Discussion by GioAc96** [mia-platform/community#436] [Product Feedback]
### What product would you like to leave feedback for?

Console

### What Console version are you using?

12

### Description

While loading the Variables page of a project, inside the "Overview" section, the pagination component temporarily shows a "NaN" value and a number of pages that is always set to 5. As soon as the variables load, the component shows the appropriate values. See the attached screenshot:

![image](https://github.com/mia-platform/community/assets/21307773/abe55256-5879-49c7-9b51-f2f76df9633d)

**Comment by fredmaggiowski** [comment]
Hi, seems like a UI glitch, thanks for reporting!
